app/src/main/java/com/example/coursematchdaddy/clean_architecture_layers/entities/interfaces/course_class_imports_implementations/ArtsAndSciencesCourseInterface.java

```diff
@@ -1,4 +1,8 @@
 package com.example.coursematchdaddy.clean_architecture_layers.entities.interfaces.course_class_imports_implementations;
 
+import com.example.coursematchdaddy.clean_architecture_layers.entities.classes.Course;
+import com.example.coursematchdaddy.clean_architecture_layers.entities.classes.User;
+
 public interface ArtsAndSciencesCourseInterface {
+    public boolean enroll(User user);
```

Comment:
Update the design doc to reflect the fact that you've added this method to the interface